app/src/main/java/com/gathering/android/auth/password/ResetPassInfoBottomSheet.kt

```diff
@@ -1,32 +0,0 @@
-package com.gathering.android.auth.password
-
-import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import androidx.fragment.app.DialogFragment
-import com.gathering.android.R
-import com.gathering.android.databinding.BottomSheetForgetPassInfoBinding
-import com.google.android.material.bottomsheet.BottomSheetDialogFragment
-import dagger.hilt.android.AndroidEntryPoint
-
-@AndroidEntryPoint
-class ResetPassInfoBottomSheet : BottomSheetDialogFragment() {
-
-    private lateinit var binding: BottomSheetForgetPassInfoBinding
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setStyle(
```

Comment:
this shouldnt have been deleted for the view to work just as before when compose is disabled. when introducing a feature flag, not a single line of code should change in case the feature flag is off